README.md

```diff
@@ -91,6 +91,19 @@ Default setup uses CPU (`--ngpu 0`).
 
 Note that if you want to use multi-gpu, the installation of nccl is required before setup.
 
+### Docker Container
+For working inside a docker container execute `run.sh` located inside the docker folder, it will builds a container and execute the given applicaiton. The execution line is detailed as following: 
+```sh
+$ cd $ESPnet_Folder/docker
+$ ./run.sh [--docker_gpu 0 --docker_egs chime4 --docker_folders /export/corpora4/CHiME4/CHiME3] --dlayers 1 --ngpu 1 
+```
+The arguments required for the docker configuration are detailed inside brackets. DO NOT add brackets at execution. Any additional configuration should be included after the previous.
```

Comment:
> The arguments required for the docker configuration are detailed inside brackets. DO NOT add brackets at execution. Any additional configuration should be included after the previous.

The arguments required for the docker configuration have a prefix "--docker" (e.g., `--docker_gpu`, `--docker_egs`, `--docker_folders`). `run.sh` accept all normal ESPnet arguments, which must be followed by these docker arguments.